docker-compose.yaml

```diff
@@ -0,0 +1,89 @@
+version: "3"
+
+services:
+  orodha-gateway:
+    hostname: orodha-gateway
+    container_name: orodha-gateway
+    image: nginx:latest
+    volumes:
+      - ./config/gateway/conf.d:/etc/nginx/conf.d
+    ports:
+      - "80:80"
+    networks:
+      - orodha-network
+
+  orodha-user-server:
+    hostname: user-service-server
+    container_name: user-service
+    image: edisonstuart/orodha-user-service:latest
```

Comment:
@Edison-Stuart May want to drive in the version for each service as part of the env, e.g. `NOTIFICATIONS_SERVICE_VERSION`, `USER_SERVICE_VERSION`, etc.